support-frontend/tsconfig.json

```diff
@@ -13,7 +13,8 @@
 		"noImplicitAny": true, // Enable error reporting for expressions and declarations with an implied any type
 		"baseUrl": "./assets", // The baseUrl to be used when resolving absolute module paths
 		"paths": {
-			"ophan": ["../node_modules/ophan-tracker-js/build/ophan.support"]
+			"ophan": ["../node_modules/ophan-tracker-js/build/ophan.support"],
```

Comment:
side note: what's this ophan thing doing?

Reply:
It's a variant of Ophan's tracker-js module tailored for the support site (https://github.com/guardian/ophan/tree/main/tracker-js), we use the module for running Ophan in the client and tracking different events.